templates/arm-aks-template.json

```diff
@@ -17,6 +17,14 @@
       "type": "int",
       "defaultValue": 2
     },
+    "nodeMinCount": {
+      "type": "int",
+      "defaultValue": 2
```

Comment:
would prefer that this was just explicitly set, should be easy if this is extracted out to the arm parameters file:
https://github.com/hmcts/cnp-aks-rbac-pipelines/tree/master/templates/vars

Note may need to update the pipeline to use the vars, it's possible only the vnet template is using it atm